pep/app/constants/regex.ts

```diff
@@ -11,4 +11,5 @@ export const NON_DIGIT_REGEX = /\D/g;
 export const SEARCH_STRING_REGEX = /\(([^()]+)\)/g;
 export const SEARCH_STRING_TERMS_REGEX = /[^a-zA-Z\s-]+/g;
 export const URL_REGEX = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/;
-export const MEDIA_FILE_EXTENSION = /\.[^/.]+$/;
+export const MEDIA_FILE_EXTENSION_REGEX = /\.[^/.]+$/;
+export const BOOLEAN_OPERATOR_REGEX = /\sOR\s|\sAND\s/;
```

Comment:
per his comment doesnt this also need to allow lowercase and `||` / `&&`?

```suggestion
export const BOOLEAN_OPERATOR_REGEX = /\sOR\s|\sAND\s|\s\|\|\s|\s\&\&\s/i;
```

Reply:
Yes, seems like he changed his comment after I read it 😭